### PR TITLE
Fix database read error causes overwrite on start

### PR DESCRIPTION
**Pull request by JonathanPorta**
- Relax logic for error check to be any non-falsey value.
- Add metadata file to .gitignore and remove from repo.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 node_modules/
 dist/
 npm-debug.log
+.DS_Store
```
Comment (DavidSouther) [inline .gitignore]: Remove, and add to your global gitignore.

File: src/api/record/record.ts [hunk 2]
```diff
@@ -28,9 +27,8 @@ export class RecordHandler extends RupertPlugin {
     private database: RecordDatabase = {}
   ) {
     super();
-    this.cancelWrite = setInterval(() => this.write(), 1 * 1000);
     readFile(this.dbPath, 'utf-8', (err: any, persisted: string) => {
-      if ( err !== null ) { return; }
+      if ( err ) { return; }
```
Comment (DavidSouther) [inline src/api/record/record.ts]: We should probably log at `info` that we didn't find a database, and are starting a new one.

File: .gitignore (hunk 1 above)
Comment (JonathanPorta) [inline .gitignore]: Oh yeah, I always forget about that.

**Comment by DavidSouther** [comment]
LGTM

File: src/api/record/record.ts (hunk 2 above)
Comment (JonathanPorta) [inline src/api/record/record.ts]: @DavidSouther Can I get an example usage for the logger?

Comment (DavidSouther) [inline src/api/record/record.ts]: https://github.com/DavidSouther/rupert/blob/ts-master/src/plugins/static.ts#L19

File: src/api/record/record.ts [hunk 3]
```diff
@@ -23,14 +23,16 @@ export class RecordHandler extends RupertPlugin {
...
     @Optional()
     @Inject(RecordDatabase)
     private database: RecordDatabase = {}
   ) {
     super();
-    this.cancelWrite = setInterval(() => this.write(), 1 * 1000);
     readFile(this.dbPath, 'utf-8', (err: any, persisted: string) => {
-      if ( err !== null ) { return; }
+      if ( err ) {
+        return logger.info('Could not load database', this.dbPath);
```
Comment (DavidSouther) [inline src/api/record/record.ts]: Should be a single string, and doesn't need to be this harsh. 
```
logger.info(`No database found, creating a new one at ${this.dbPath}`);
return;
```

**Comment by DavidSouther** [comment]
LGTM

**Pull request merged**